packages/css/src/unordered-list/unordered-list.scss

```diff
@@ -29,7 +38,24 @@
   @include reset;
 }
 
-.amsterdam-unordered-list__item {
+.amsterdam-unordered-list > .amsterdam-unordered-list__item {
```

Comment:
Why was the `>` selector necessary again? An `unordered-list__item` always has an `unordered-list` as a parent, no?

Reply:
Otherwise, list items of a nested list will be affected. The intention is to aim for the items of this specific list only, the selector should express that.

Reply:
The current selector doesn't meet that goal though, I think? Nested lists have the following class structure:

```css
amsterdam-unordered-list
  unordered-list__item <- this is selected
  amsterdam-unordered-list
    unordered-list__item <- but this is selected as well
```

Reply:
Yes but that is accidental inheritance, right? Compare how we also assign a font family to an inline link.

Additionally, if we want to support mixing ordered and unordered lists, we need to make sure nothing gets mixed up between the two variations of list item.

Come to think of it, shouldn’t we have `amsterdam-ordered-list__item` and `amsterdam-unordered-list__item`?

Reply:
Discussed and agreed. We’ll improve this soon.